pkg/apis/kops/instancegroup.go

```diff
@@ -109,6 +109,16 @@ type InstanceGroupSpec struct {
 	Kubelet *KubeletConfigSpec `json:"kubelet,omitempty"`
 	// Taints indicates the kubernetes taints for nodes in this group
 	Taints []string `json:"taints,omitempty"`
+	// IAMProfileSpec defines the identity of the cloud group iam profile.
+	IAM *IAMProfileSpec `json:"iam,omitempty"`
+}
+
+// IAMProfileSpec defined the identity of the cloud group iam profile.
```

Comment:
Present tense I think i.e. defines, not defined.

But how about "IAMProfileSpec contains the IAM related settings for the instance group"